### ISSUE TITLE
Microsoft.FluentUI.AspNetCore.Components does not have a strong name

### ISSUE THREAD

**Issue by jjzhang12** [microsoft/fluentui-blazor#1677]
# 🐛 Bug Report

Consume nuget package Microsoft.FluentUI.AspNetCore.Components 4.5.0 or 4.5.1-preview.24068.9 to blazor web assembly project and try to publish, the visual studio reports
"Referenced assembly 'Microsoft.FluentUI.AspNetCore.Components, Version=4.5.1.24068, Culture=neutral, PublicKeyToken=null' does not have a strong name."
and publish failed.

## 🤔 Expected Behavior

Visual studio should publish the project with the dll, and the dll should have strong name.

## 😯 Current Behavior
Visual studio failed to publish the project with the dll because it does not have strong name.

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* .NET and Fluent UI Blazor library Version [e.g. 8.0.2 and 4.4.1]: Microsoft.FluentUI.AspNetCore.Components 4.5.0, Visual Studio 2022 17.9.2, .net 8.

**Comment by vnbaaij** [comment]
Not sure what you mean with a strong name. 

Only our 'normal' versions ( for example 4.5.0) are published assigned packages. Preview packages are just for preview, are not signed and are not meant to be used in published applications.

**Comment by jjzhang12** [comment]
Hi @vnbaaij : the Microsoft.FluentUI.AspNetCore.Components.dll in 4.5.0 and earlier versions (I tried 4.3.0) aslo not signed. I use "sn -v" and it reports "Microsoft.FluentUI.AspNetCore.Components.dll does not represent a strongly named assembly" as well.

**Comment by vnbaaij** [comment]
The packages are signed (with a valid certificate). Following the guidance in https://github.com/dotnet/runtime/blob/main/docs/project/strong-name-signing.md#1-microsoft-strong-names-their-assemblies-should-i we don't plan on signing/giving a strong name to the individual files.

**Comment by jjzhang12** [comment]
The FAQ did say "Because of the viral nature of strong-naming, it is recommended that publicly published .NET libraries are strong-named.". Without strong name, any strong named project cannot consume it.

**Comment by vnbaaij** [comment]
True, but as said it is a relic from the .NET Framework past. Following the link from the FAQ to https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/strong-naming#create-strong-named-net-libraries, it says there

>You should strong name your open-source .NET libraries if their targets include .NET Framework or .NET Standard. Strong naming is not required for libraries that target .NET Core/5+ only.

As a Microsoft project we are following that. As said, we are not planning on adding strong names to our project. In case a referencing project is using strong naming, I would say (following the guidance) to remove strong naming from that project as well.

**Comment by vnbaaij** [comment]
Closing this as not planned
